src/collections/_documentation/development/sdk-dev/overview.md

```diff
@@ -77,7 +77,8 @@ Sentry.init({
 ```
 
 {% capture __alert_content -%}
-If an empty DSN is passed, you should treat it as valid option which signifies disabling the SDK.
+If an empty DSN is passed, you should treat it as valid option which signifies not sending any events to the server.
+The SDK will however still process events and submit them to the configured transport.
```

Comment:
What is the advantage of passing event to the transport if the SDK is disabled?
Does this need to be a rule? I've taken the line above quite literally and I have looked up for a DSN on init:

https://github.com/getsentry/sentry-dotnet/blob/master/src/Sentry/SentrySdk.cs#L79-L92

If no DSN exists, I keep a NoOp hub bound to the static accessor.

Reply:
I think its a valid usecase if you have a custom transport configured which does not depend on a DSN.

Reply:
Could be a valid use-case even though it hasn't come up yet but honestly this sounds like an edge case and maybe better just left off of the spec.